armi/operators/operator.py

```diff
@@ -376,10 +385,14 @@ def _timeNodeLoop(self, cycle, timeNode):
         self.r.p.timeNode = timeNode
         self.interactAllEveryNode(cycle, timeNode)
         # perform tight coupling if requested
-        if self.cs["numCoupledIterations"]:
-            for coupledIteration in range(self.cs["numCoupledIterations"]):
+        if self.cs["tightCoupling"]:
+            for coupledIteration in range(self.cs["tightCouplingMaxNumIters"]):
                 self.r.core.p.coupledIteration = coupledIteration + 1
+                self._tightCouplingOldValues()
```

Comment:
method name should be a verb. What is it doing? recording them?

Also, I prefer `_somethingTightCouplingPreviousValues` rather than `Old` b/c I think it is more clear that they're the previous iteration's values rather than some old antiquated values from a previous code version or something